packages/deployer/src/utils/compiler.ts

```diff
@@ -105,16 +104,15 @@ export function parseDependencies(source: string): string[] {
 
 /**
  * Callback to resolve dependencies with `solc.compile`.
- * Throws error if contractSources not yet initialized.
  * @param  contractSources Source codes of contracts.
  * @param  importPath Path to an imported dependency.
  * @return Import contents object containing source code of dependency.
  */
 export function findImportIfExist(contractSources: ContractSources, importPath: string): solc.ImportContents {
```

Comment:
Can we just pass in `contractSource` instead of `contractSources`?

Reply:
This wil be out after the resolver PR